Automattic/woocommerce-payments
# Resolve merge conflicts between the latest develop and split UPE branch

#### Changes proposed in this Pull Request
This PR resolves conflicts between `develop` and `poc/upe-instances-multiplied` and aligns the existing tests (9547544f24f4a36441eb8341824666f89989a909) so that the new tests added a few days ago are compatible with the split UPE implementation. The conflicts were resolved in:

```
#       client/checkout/classic/upe.js
#       includes/constants/class-payment-method.php
#       includes/payment-methods/class-upe-payment-gateway.php
#       tests/unit/admin/test-class-wc-rest-payments-settings-controller.php
#       tests/unit/payment-methods/test-class-upe-payment-gateway.php
```










#### Testing instructions
* Ensure that the WooCommerce Payments method in `WP Admin → WooCommerce → Settings → Payments → All payment methods` has the same description regardless of whether the UPE is turned on or off. In a nutshell, perform the Testing Instructions [from here](https://github.com/Automattic/woocommerce-payments/pull/5394).
* Pay with UPE on the Pay for order page and ensure there are no errors during this process.
* [E2E tests](https://github.com/Automattic/woocommerce-payments/actions/runs/3968412790) should be green (except for `WC - latest | blocks - shopper` which [seem to be broken on `develop` too](https://github.com/Automattic/woocommerce-payments/actions/runs/3959696512)).
* Perform one purchase with any UPE method with shortcode and Blocks checkouts and ensure that the payment was captured correctly on the Stripe side and the order has the right _Processing status_ in WooCommerce.

-------------------


**Post merge**
